at-quannguyen/W4/channel.rb

```diff
@@ -0,0 +1,22 @@
+module Channel
+  def add_channel(list, *param)
+    unless list.has_key?(param[0])
+      list.merge!({param[0] => {title: param[1], content: param[2], category: param[3]}})
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/f8dbe605eafc53803b1d102f4a00dad2a5e916b9/at-quannguyen/W4/channel.rb#L4"></span>Performance/RedundantMerge: Use <code>list[param[0]] = {title: param[1], content: param[2], category: param[3]}</code> instead of <code>list.merge!({param[0] =&gt; {title: param[1], content: param[2], category: param[3]}})</code>. (https://github.com/JuanitoFatas/fast-ruby#hashmerge-vs-hash-code)</td>
    </tr>
  </tbody>
</table>